packages/@headlessui-react/src/components/combobox/combobox.tsx

```diff
@@ -1077,8 +1078,21 @@ function InputFn<
     })
   })
 
+  let dd = useDisposables()
```

Comment:
The `dd` variable here is confusing to me — what does it mean? I know we typically use `d` for disposables, so is it just because you have two disposables in this case? Wondering if we can come up with a better name here?

Reply:
Yep we already had `d` but that had a different use case. To solve this, I created a `useFrameDebounce()` hook (we can improve this name if we want). But now we can do this instead:

<img width="2391" alt="image" src="https://github.com/tailwindlabs/headlessui/assets/1834413/6188da00-e16c-4702-b11f-0afd0959f060">